platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -269,6 +269,22 @@
             ]
         )
     },
+    # 4410
```

Comment:
@vadymhlushko-mlnx why this one was added in the middle and not on the end?
BTW, this list is only for non-default systems? i dont see sn4700 and i wonder if this should be added.

Reply:
I will commit this change to the end.

SN4700 was already added - [thermal.py](https://github.com/Azure/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py) please check

Reply:
Done